packages/@aws-cdk/core/lib/stack.ts

```diff
@@ -925,8 +925,66 @@ export class Stack extends Construct implements ITaggable {
    * - You are now free to remove the `bucket` resource from `producerStack`.
    * - Don't forget to remove the `exportValue()` call as well.
    * - Deploy again (this time only the `producerStack` will be changed -- the bucket will be deleted).
+   *
+   * @deprecated Use `exportListOrStringValue`
+   */
+  public exportValue(exportedValue: any, options: ExportValueOptions = {}): string {
```

Comment:
We have to create `exportListOrString()` to avoid breakage (`exportValue()` could only return a `string`). I don't like this but I think it's the best we can do.